### DISCUSSION TITLE
Error during registration - email does not exist

### DISCUSSION THREAD

**Discussion by DarkShyMW** [mastodon/mastodon#18635] [Q&A]
Error during registration - email does not exist
I dont know whats happened.
![изображение](https://user-images.githubusercontent.com/30442729/173057183-96ff1fa9-e52c-472a-a190-b2f5fc5bebb5.png)

**Comment by Gargron** [comment]
It can't resolve DNS to mail.ru, I'm assuming

**Reply by DarkShyMW** [reply]
darkshy@bronyfurry:~$ nslookup mail.ru
Server:         1.1.1.1
Address:        1.1.1.1#53

Non-authoritative answer:
Name:   mail.ru
Address: 94.100.180.200
Name:   mail.ru
Address: 217.69.139.200
Name:   mail.ru
Address: 94.100.180.201
Name:   mail.ru
Address: 217.69.139.202
Name:   mail.ru
Address: 2a00:1148:db00:0:b0b0::1
